django_rq/views.py

```diff
@@ -144,6 +149,47 @@ def failed_jobs(request, queue_index):
     return render(request, 'django_rq/jobs.html', context_data)
 
 
+@staff_member_required
+def scheduled_jobs(request, queue_index):
+    queue_index = int(queue_index)
+    queue = get_queue_by_index(queue_index)
+
+    registry = ScheduledJobRegistry(queue.name, queue.connection)
+
+    items_per_page = 100
+    num_jobs = len(registry)
+    page = int(request.GET.get('page', 1))
+    jobs = []
+
+    if num_jobs > 0:
+        last_page = int(ceil(num_jobs / items_per_page))
+        page_range = range(1, last_page + 1)
+        offset = items_per_page * (page - 1)
+        job_ids = registry.get_job_ids(offset, offset + items_per_page - 1)
+
+        for job_id in job_ids:
+            try:
+                job = Job.fetch(job_id, connection=queue.connection)
```

Comment:
We should use `Job.fetch_many`, it's much more efficient for fetching a large number of jobs, and there's no need to handle `NoSuchJobError`

Reply:
Okay, changed fetch in `scheduled_jobs` view, did not touch other views.